tools/serverpod_cli/test/integration/analyzer/dart/endpoint_validation/endpoint_method_test.dart

```diff
@@ -120,17 +120,21 @@ class ExampleEndpoint extends Endpoint {
     });
   });
 
-  group('Given an endpoint method without Session param when analyzed', () {
+  group('Given an endpoint method without a first unnamed `Session` param', () {
```

Comment:
Suggestion: Split to two groups.

Since the setup is different for the tests, we might as well have two groups defined for them. I understand the thinking here, but when the "shared" part is not defined together for the tests in the group, it could be a bit difficult to find where it is defined.

Reply:
fixed: 
b4d9149